_includes/card-operations.md

```diff
@@ -0,0 +1,98 @@
+        "name": "4925xxxxxx000004",
+        "expiryDate" : "mm/yyyy"
+      }
+    ],
+    "details": {
+      "cardBrand": "Visa",
+      "MaskedPan": "4925xxxxxx000004",
+      "cardType": "Credit",
+      "issuingBank": "UTL MAESTRO",
+      "countryCode": "999",
+      "acquirerTransactionType": "3DSECURE",
+      "issuerAuthorizationApprovalCode": "397136",
+      "acquirerStan": "39736",
+      "acquirerTerminalId": "39",
+      "acquirerTransactionTime": "2017-08-29T13:42:18Z",
+      "nonPaymentToken" : "12345678-1234-1234-1234-1234567890AB",
+      "externalNonPaymentToken" : "1234567890",
+      "transactionInitiator" : "MERCHANT"    
+    }
+  }
+}
+```
+
+{:.table .table-striped}
+| Field                             | Type      | Description                                                                                                                                                                                                  |
+| :-------------------------------- | :-------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `payment`                         | `string`  | {% include field-description-id.md sub_resource=transaction %}                                                                                                                                               |
+| └➔&nbsp;`transaction`                     | `string`  | The transaction object, containing information about the current transaction.                                                                                                                                |
+| └─➔&nbsp;`id`                     | `string`  | {% include field-description-id.md resource=transaction %}                                                                                                                                                   | 
+| └─➔&nbsp;`number`                 | `string`  | The transaction `number`, useful when there's need to reference the transaction in human communication. Not usable for programmatic identification of the transaction, for that `id` should be used instead.                                                                                                                                                               |
+| └➔&nbsp;`payeeReference`          | `string`  | A unique reference for the transaction.                                                                                                                                                                      | 
+| └➔&nbsp;`orderReference`          | `string(50)`  | The order reference should reflect the order reference found in the merchant's systems.                                                                                                                  |                                                             
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[hard-break-spaces] Use two spaces for hard line breaks](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5705576)